featuretools/tests/entityset_tests/test_es.py

```diff
@@ -967,17 +967,50 @@ def test_normalize_entity_new_time_index_additional_success_check(es):
                         copy_variables=[])
 
 
-def test_normalize_time_index_from_none(es):
-    es['customers'].time_index = None
-    es.normalize_entity('customers', 'birthdays', 'age',
-                        make_time_index='date_of_birth',
-                        copy_variables=['date_of_birth'])
-    assert es['birthdays'].time_index == 'date_of_birth'
-    df = es['birthdays'].df
+@pytest.fixture
+def pd_normalize_es():
+    df = pd.DataFrame({
+        "id": [0, 1, 2, 3],
+        "A": [5, 4, 2, 3],
+        'time': [datetime(2020, 6, 3), (datetime(2020, 3, 12)), datetime(2020, 5, 1), datetime(2020, 4, 22)]
+    })
+    es = ft.EntitySet("es")
+    return es.entity_from_dataframe(
+        entity_id="data",
+        dataframe=df,
+        index="id")
+
+
+@pytest.fixture
+def dd_normalize_es(pd_normalize_es):
+    es = ft.EntitySet(id=pd_normalize_es.id)
+    for entity in pd_normalize_es.entities:
```

Comment:
we can remove this for loop since we know there is only one entity

Reply:
Ended up doing `entity = pd_normalize_es.entities[0]`, but I played around with some other ways of doing this that just build the entity set from the dataframe directly, and none of them worked with the dask test. This might be in me not understanding something about either dask or fixtures, but I'm not sure why something like this doesn't work:

```
@pytest.fixture
def normalize_df():
    return pd.DataFrame({
        "id": [0, 1, 2, 3],
        "A": [5, 4, 2, 3],
        'time': [datetime(2020, 6, 3), (datetime(2020, 3, 12)), datetime(2020, 5, 1), datetime(2020, 4, 22)]
    })


@pytest.fixture
def pd_normalize_es(normalize_df):
    df = normalize_df
    es = ft.EntitySet("es")
    return es.entity_from_dataframe(
        entity_id="data",
        dataframe=df,
        index="id")


@pytest.fixture
def dd_normalize_es(normalize_df):
    es = ft.EntitySet("es")
    return es.entity_from_dataframe(
        entity_id="data",
        dataframe=dd.from_pandas(normalize_df, npartitions=2),
        index="id")

@pytest.fixture(params=['dd_normalize_es', 'pd_normalize_es'])
def normalize_es(request):
    return request.getfixturevalue(request.param)
```

Reply:
It might be that dask entities need the variable types for each column set in `entity_from_dataframe` to avoid the cost of trying to infer them.